### ISSUE TITLE
Ligature at end of section masked out

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#336]
A ligature at the end of a section, e.g.
```
\section{Stuff}
```
is somehow masked out when the section is selected:
<img width="150" alt="screen shot 2018-11-11 at 13 57 41" src="https://user-images.githubusercontent.com/650763/48313285-3aeba680-e5ba-11e8-8c75-f6fffc5e7b55.png">

This only seems to occur if the ligature is at the end of the section:
```
\section{Stuffing}
```
works fine
<img width="181" alt="screen shot 2018-11-11 at 14 02 06" src="https://user-images.githubusercontent.com/650763/48313294-68385480-e5ba-11e8-9463-8585e6751e54.png">

**Comment by ktraunmueller** [comment]
Here's a test document:
```
\documentclass[12pt,a4paper]{article}

\usepackage[utf8]{inputenc}
\usepackage{geometry}
\usepackage{lmodern}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\graphicspath{{./}}
\usepackage{csquotes}

\begin{document}

\title{Ligatures}

\maketitle

\section{Stuff}

There you have it.

\end{document}
```

**Comment by ktraunmueller** [comment]
Fixed in 1.24
